content/security/faqs/single-sign-on/idp-faqs.md

```diff
@@ -51,16 +47,16 @@ Yes, bot accounts need a seat, similar to a regular end user, having a non-alias
 > Optional Just-in-Time (JIT) provisioning configuration is available in [beta](/release-lifecycle/#beta) when you use the Admin Console and enable SCIM. Otherwise, JIT is enabled by default.
 { .experimental }
 
-The SSO implementation uses Just-in-Time (JIT) provisioning by default. You can optionally disable JIT if you prefer not to auto-provision users, or if you opt for auto-provisioning using SCIM.
+The SSO implementation uses Just-in-Time (JIT) provisioning by default. You can optionally disable JIT if you prefer not to auto-provision users, or if you opt for auto-provisioning using SCIM. See [Just-in-Time provisioning](/security/for-admins/provisioning/just-in-time/).
 
-### Will there be IdP-initiated logins?
+### Is IdP-initiated sign-in available?
 
-We currently don't have any plans to enable IdP-initiated logins.
+Docker SSO doesn't support IdP-initiated sign-in, only Service Provider Initiated (SP-initiated) sign-in.
 
 ### Is it possible to connect Docker Hub directly with a Microsoft Entra (formerly Azure AD) group?
 
 Yes, Entra ID (formerly Azure AD) is supported with SSO for Docker Business, both through a direct integration and through SAML.
 
-### My SSO connection with Entra ID (formerly Azure AD) isn't working and I receive an error that the application is misconfigured. How can I troubleshoot this?
+### My SSO connection with Entra ID isn't working and I receive an error that the application is mis-configured. How can I troubleshoot this?
```

Comment:
Maybe it's just me, but I actually prefer "misconfigure" over "mis-configure"

```suggestion
### My SSO connection with Entra ID isn't working and I receive an error that the application is misconfigured. How can I troubleshoot this?
```

Reply:
I do too but vale yelled at me 😅 I think I'll change it back